src/modules/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.22)
+
+project(ModuleManager)
+
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Check if you can use `set_common_settings` here (`src/cmake/CommonSettings.cmake`).